Source/RunActivity/Viewer3D/Popups/HelpWindow.cs

```diff
@@ -1152,6 +1183,231 @@ private void writeline()
+        /// Loops through the train cars and aggregates the info.
+        /// </summary>
+        private void AddAggregatedTrainInfo(Train playerTrain, ControlLayout scrollbox, int labelWidth)
+        {
+            string numEngines = ""; // using "+" between DPU sets
+            int numCars = 0;
+            int numAxles = 0;
+            var totMassKg = playerTrain.MassKg;
+            string sectionMass = ""; // using "  +  " between wagon sets
+            var lengthM = playerTrain.Length;
+            var maxSpeedMps = playerTrain.TrainMaxSpeedMpS;
+            float totPowerW = 0f;
+            float totMaxTractiveEffortN = 0f;
+            string sectionMaxTractiveForce = ""; // using "  +  " between DPU sets
+            float totMaxDynamicBrakeForceN = 0f;
+            string sectionMaxDynamicBrakeForce = ""; // using "  +  " between DPU sets
+            float maxBrakeForceN = 0f;
+            float lowestCouplerStrengthN = 9.999e8f;  // impossible high force
+            float lowestDerailForceN = 9.999e8f;  // impossible high force
+
+            int engCount = 0; string countSeparator = ""; // when set, indicates that subsequent engines are in a separate block
+            float engMaxTractiveForceN = 0f; float engMaxDynBrakeForceN = 0f; string forceSeparator = "";  // when set, indicates that subsequent engines are in a separate block
+
+            float wagMassKg = 0f; string massSeparator = ""; // when set, indicates that subsequent engines are in a separate block
+            int numOperativeBrakes = 0;
+            bool isMetric = false; bool isUK = false;  // isImperial* does not seem to be used in simulation
+
+            if (TrainInfoSpriteSheet == null) { TrainInfoSpriteSheet = SharedTextureManager.Get(Owner.Viewer.RenderProcess.GraphicsDevice, Path.Combine(Owner.Viewer.ContentPath, "TrainInfoSprites.png")); }
+            const int spriteWidth = 6; const int spriteHeight = 26;
+            var carInfoList = new List<CarInfo>(playerTrain.Cars.Count);
+
+            foreach (var car in playerTrain.Cars)
+            {
+                // ignore (legacy) EOT
+                if (car.WagonType == TrainCar.WagonTypes.EOT || car.CarLengthM < 1.1f) { continue; }
+
+                var wag = car is MSTSWagon ? (MSTSWagon)car : null;
+                var eng = car is MSTSLocomotive ? (MSTSLocomotive)car : null;
+
+                var isEng = (car.WagonType == TrainCar.WagonTypes.Engine && eng != null && eng.MaxForceN > 25000);  // count legacy driving trailers as wagons
+
+                if (car.IsMetric) { isMetric = true; }; if (car.IsUK) { isUK = true; }
+
+                if (isEng)
+                {
+                    engCount++;
+                    numAxles += eng.LocoNumDrvAxles + eng.GetWagonNumAxles();
+                    totPowerW += eng.MaxPowerW;
+                    totMaxTractiveEffortN += eng.MaxForceN;
+                    engMaxTractiveForceN += eng.MaxForceN;
```

Comment:
For the purposes of what you are doing here, you don't want the total _maximum_ tractive effort, you want the total _continuous_ tractive effort.

```suggestion
                    totMaxTractiveEffortN += eng.MaxContinuousForceN > 0 ? eng.MaxContinuousForceN : eng.MaxForceN;
                    engMaxTractiveForceN += eng.MaxContinuousForceN > 0 ? eng.MaxContinuousForceN : eng.MaxForceN;
```
(The > 0 check is needed as the max continuous force is not a required input and may be left at 0)

Knowing the maximum tractive effort isn't helpful if that level of effort would require frying the motors. Hence, all of UP's rated power axles are defined based on the continuous rating to ensure nobody makes a train that can only temporarily give enough tractive effort for the job at hand.

A good example of that is the current UP SSI giving data for their E9s. Because of the extreme gear ratio used on E units, the power rating for the E9 is just 3.5 despite having 4 updated motors and enough weight to get at least 55,000 pounds down without slipping. Anything more than around 35,000 pounds will overheat the motors as a lot of amperage is needed to achieve that traction.

Reply:
Done. I also renamed the variables to reflect the change.

I will do the same in the Content Manager.